### ISSUE TITLE
Anchor: target checkbox bad positioning (4.12.x branch) ?

### ISSUE THREAD

**Issue by brazorf** [yabwe/medium-editor#882]
```javascript
new MediumEditor(".editor", {
    buttons: ['italic', 'bold', 'underline', 'anchor', 'unorderedlist', 'quote'],
    anchorTarget: true,
    anchorInputCheckboxLabel: "Target: blank"
});
```

The checkbox display on the far right and the relative label on the bottom.
[See plunker here](http://plnkr.co/edit/dJcLzvCdprGEzpTVEhLH?p=preview)

**Comment by nmielnik** [comment]
This was fixed via https://github.com/yabwe/medium-editor/commit/d40c6ed9bdb55df9b1ca9e9344d95008f1d121f0

I'll port this commit over to the 4.12.x branch.

**Comment by nmielnik** [comment]
I've ported over the fix from master and cut a new release of 4.12.12 which will include the fix for this.

I would definitely recommend trying to stick with version 5.x, our 4.12.x branch is getting further and further behind master as we speak :worried:

**Comment by brazorf** [comment]
I would like to stick with latest, but i'm currently using an angular module that is bound to an outdated version :(
